apps/main/src/llamalend/features/user-position-history/RowExpandedPanel.tsx

```diff
@@ -0,0 +1,70 @@
+import ArrowOutwardIcon from '@mui/icons-material/ArrowOutward'
+import Button from '@mui/material/Button'
+import Stack from '@mui/material/Stack'
+import Typography from '@mui/material/Typography'
+import { formatNumber } from '@ui/utils'
+import { t } from '@ui-kit/lib/i18n'
+import { type ExpandedPanel } from '@ui-kit/shared/ui/DataTable/ExpansionRow'
+import { SizesAndSpaces } from '@ui-kit/themes/design/1_sizes_spaces'
+import type { ParsedUserCollateralEvent } from './hooks/useUserCollateralEvents'
+
+const { Spacing } = SizesAndSpaces
+
+export const RowExpandedPanel: ExpandedPanel<ParsedUserCollateralEvent> = ({ row: { original: event } }) => {
+  const { txUrl, loanChange, borrowToken, collateralChange, collateralChangeUsd, collateralToken } = event
+
+  const handleViewTransaction = () => {
+    window.open(txUrl, '_blank')
+  }
+
+  return (
+    <Stack>
+      <Stack paddingTop={Spacing.md}>
+        <Stack direction="row" justifyContent="space-between">
+          <Typography
+            variant="bodyMRegular"
+            sx={(t) => ({ color: t.design.Text.TextColors.Secondary })}
+          >{t`Collateral`}</Typography>
+          <Typography
+            variant="tableCellMBold"
+            color={
+              collateralChange === 0 || collateralChange == null
+                ? 'textPrimary'
+                : collateralChange > 0
+                  ? 'success'
+                  : 'error'
+            }
+          >
+            {collateralChange > 0 ? '+' : ''}
+            {collateralChange === 0 ? '-' : formatNumber(collateralChange)}{' '}
+            {collateralChange != null && collateralChange !== 0 && collateralChangeUsd !== 0 && collateralToken?.symbol}
+          </Typography>
+        </Stack>
+        <Stack direction="row" justifyContent="space-between">
+          <Typography
+            variant="bodyMRegular"
+            sx={(t) => ({ color: t.design.Text.TextColors.Secondary })}
+          >{t`Debt`}</Typography>
+          <Typography
+            variant="tableCellMBold"
+            color={loanChange === 0 || loanChange == null ? 'textPrimary' : loanChange > 0 ? 'error' : 'success'}
+          >
+            {loanChange > 0 ? '+' : ''}
+            {loanChange !== 0 ? formatNumber(loanChange) : '-'}{' '}
+            {loanChange != null && loanChange !== 0 && borrowToken?.symbol}
+          </Typography>
+        </Stack>
+      </Stack>
+
+      <Button
```

Comment:
Instead of `onClick` check how to make the button a `<a>` component that opens a link on a diff tab, see the disclaimer code:

```ts
      <Button
        component={Link}
        variant="link"
        href="https://docs.curve.finance/references/whitepaper"
        target="_blank"
        endIcon={<ArrowOutwardIcon />}
        sx={{
          padding: Spacing.xs,
          '&': { fontWeight: '500' },
        }}
      >
        {t`Curve whitepaper`}
      </Button>
```